src/backend/api/reservations.js

```diff
@@ -0,0 +1,95 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
```

Comment:
You can create a table like `Reservation` before the end points are available. This would help with testing as well.
`
knex.schema.hasTable('Reservation').then(function(exists) {
  if (!exists) {
    return knex.schema.createTable('Reservation', function (table) {
      table.integer('id').primary();
      table.dateTime('created_date');
      table.string('contact_phonenumber');
      table.string('contact_name');
      table.string('contact_email');
      table.integer('number_of_guests');
      table.integer('meal_id');
    });
  }
});
`